src/angular/components/set-password.component.ts

```diff
@@ -20,6 +21,7 @@ import { ChangePasswordComponent as BaseChangePasswordComponent } from './change
 import { KdfType } from '../../enums/kdfType';
 
 export class SetPasswordComponent extends BaseChangePasswordComponent {
+    loadingSync: boolean = true;
```

Comment:
Shouldnt you be setting this to false after sync?

Reply:
Yep - it didn't make it into the commit for some reason. Updated.